commons/addons/cellar-vs-fsbuckets.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Choosing between File System Buckets and Cellar S3
+position: 3
+shortdesc: This will help you understand the differenc between Cellar and FS Buckets.
+tags:
+- addons
+keywords:
+- S3
+- amazon
+- Storage
+- file
+- files
+---
+
+You want to store files but you can't decide between FS Buckets and Cellar. This article
+will compare them: usage, pros/cons, pricing. Get ready for the ultimate comparison.
```

Comment:
> ultimate

avoid superlative in documentation, it's not marketing

Reply:
Fixed. Removed the "sensational" sentence ;)